railties/test/generators/api_app_generator_test.rb

```diff
@@ -94,6 +94,18 @@ def test_generator_if_skip_action_mailer_is_given
     assert_no_directory "app/views"
   end
 
+  def test_generator_skip_css
+    run_generator [destination_root, "--api", "--css=tailwind"]
+
+    assert_file "Gemfile" do |content|
+      assert_no_match(%r/gem "tailwindcss-rails"/, content)
+    end
+
+    assert_no_file "app/views/layouts/applicaearion.html.erb" do |content|
+      assert_no_match(/tailwind/, content)
+    end
```

Comment:
The filename here is misspelled. Additionally, since the expectation is that there is no file I don't think the block is ever run.